students/scottebetts/lesson04/mailroom.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+from time import strftime
+
+donors = [['Tim Berners-Lee', 120000.00, 3, 40000.00],
+          ['Ada Lovelace', 600.00, 2, 300.00],
+          ['Marie Curie', 1000.00, 2, 500.00],
+          ['Nicola Tesla', 20000.00, 4, 5000.00],
+          ['Hans Holbein', 300.00, 3, 100.00]]
+
+
+def menu_selection(prompt, dispatch_dict):
+    while True:
+        response = input(prompt)
+        if dispatch_dict[response]() == "exit menu":
+            break
+
+
+def thanks_menu():
+    menu_selection(thanks_prompt, thanks_dispatch)
+
+
+def quit():
+    print("Quitting this menu now")
+    return "exit menu"
```

Comment:
I would recommend to use `sys.exit()` here instead of return statement then your main loop becomes even simpler